Resources/Prototypes/_EE/Entities/NanoTrasen/Modsuits/inquisitory.yml

```
@@ -0,0 +1,163 @@
+      walkModifier: 1
+      sprintModifier: 1
+    - type: ToggleClothing
+      action: ActionToggleMagboots
+    - type: ComponentToggler
+      components:
+        - type: NoSlip
+    - type: Magboots
+    - type: SealableClothingVisuals
+      visualLayers:
+        shoes:
+        - state: equipped-FEET-sealed
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)